omnibus/config/software/datadog-agent-integrations-py3.rb

```diff
@@ -287,6 +286,20 @@
         shellout! "patchelf --add-rpath #{install_dir}/embedded/lib #{so_to_patch}"
         FileUtils.rm([libssl_match, libcrypto_match])
       end
+
+      block "Patch psycopg's openssl linking" do
+        # Same for psycopg
+        psycopg_folder = "#{site_packages_path}/psycopg_c"
+        libssl_match = Dir.glob("#{psycopg_folder}.libs/libssl-*.so.3")[0]
+        libcrypto_match = Dir.glob("#{psycopg_folder}.libs/libcrypto-*.so.3")[0]
```

Comment:
I guess `glob`s are so we don't have to hardcode the versions of these libraries, correct?
Shouldn't we at least check that one and only one is found in each case?

Reply:
That's correct, 
- For the ones pointed at by this comment, they have a hash attached to them, and we don't want the burden of updating them every time. And in this case since we're accessing the `[0]` element, this would fail if they weren't found.
- For the ones right below, some of the file names are dependent on architecture and python version. It could probably be tightened a bit to make it more explicit which files are being included. I'd like to do that as a follow-up, though, given that these changes have been tested in their current state (with deploys, which is a costly thing in terms of feedback time) and we want to get this released to solve the incident's bug.